src/components/Header.js

```diff
@@ -21,7 +21,7 @@ function Header() {
           onMouseEnter={(e) => e.target.style.transform = 'scale(1.1)'} // Увеличение при наведении
           onMouseLeave={(e) => e.target.style.transform = 'scale(1)'}   // Возврат к исходному размеру
         >
-          GET A HUG
+          BECOME A CHAMPION
```

Comment:
what does it mean?

better 'get my programm' or 'apply for program'

Reply:
name 'BECOME A CHAMPION' is ambiguous...not clear what it would do

actions (button) must be clear for user... life is hard on it's own

Reply:
so you moved it to Header component

got you 👍